pkg/platform/cicd/service.go

```diff
@@ -0,0 +1,116 @@
+package cicd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/dolittle-entropy/platform-api/pkg/platform"
+	"github.com/dolittle-entropy/platform-api/pkg/utils"
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+type service struct {
+	logContext      logrus.FieldLogger
+	k8sDolittleRepo platform.K8sRepo
+}
+
+func NewService(logContext logrus.FieldLogger, k8sDolittleRepo platform.K8sRepo) *service {
+	s := &service{
+		logContext:      logContext,
+		k8sDolittleRepo: k8sDolittleRepo,
+	}
+	return s
+}
+
+func (s *service) GetServiceAccountCredentials(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	applicationID := vars["applicationID"]
+	userID := r.Header.Get("User-ID")
+	tenantID := r.Header.Get("Tenant-ID")
+
+	allowed := s.k8sDolittleRepo.CanModifyApplicationWithResponse(w, tenantID, applicationID, userID)
+	if !allowed {
+		return
+	}
+
+	logContext := s.logContext.WithFields(logrus.Fields{
+		"method":      "GetServiceAccountCredentials",
+		"credentials": "serviceAccount",
+		"userID":      userID,
```

Comment:
```suggestion
		"userID":      userID,
		"applicationID": applicationID,
		"customerID": tenantID,
```

More values in the logs the merrier right?

Reply:
Done.